app/src/main/java/helium314/keyboard/keyboard/KeyboardActionListener.java

```diff
@@ -97,6 +97,7 @@ public interface KeyboardActionListener {
      */
     boolean onHorizontalSpaceSwipe(int steps);
     boolean onVerticalSpaceSwipe(int steps);
+    void resetSubtypeSwitchCount();
```

Comment:
Could you rename it to something more generic like `onEndSpaceSwipe`?
There is no neet to tie a function that's called on ending every space swipe to a specific purpose.